src/components/Check/OptionRate.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+// @ts-ignore
+import { GU, useTheme, useLayout } from '@aragon/ui'
+import { fontWeight } from '../../style/font'
+import { radius } from '../../style/radius'
+import { shadowDepth } from '../../style/shadow'
+import TokenGraphic from '../TokenGraphic/TokenGraphic'
+import styled, { css } from 'styled-components'
+import { Rate } from '../Migrate/Converter/ConversionRate'
+import Tooltip from './Tooltip'
+import { useOptionConversionRate } from '../../hooks/usePolledBalance'
+
+type OptionRateProps = {
+  compactMode: boolean
+  tokenSymbol: string
+}
+
+function OptionRate({ tokenSymbol }: OptionRateProps): JSX.Element {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small' || layoutName === 'medium'
+  const conversionRate = useOptionConversionRate()
+
+  return (
+    <div>
+      <Pill color={theme.surface}>
+        <TokenGraphic tokenName={'opt'} shadow size={compactMode ? 80 : 100} />
+        <TokenGraphic
+          tokenName="antV2"
+          size={compactMode ? 80 : 100}
+          shadow
+          css={`
+            margin-left: -${1 * GU}px;
+          `}
+        />
+      </Pill>
+      <TextContainer>
+        <h4
+          css={`
+            font-size: ${compactMode ? 52 : 60}px;
+            font-weight: ${fontWeight.semiBold};
+
+            // Optically offset left edge of number 1 for centering
+            margin-left: -0.15em;
+            letter-spacing: -0.075em;
+
+            line-height: 1;
+            margin-bottom: ${1.75 * GU}px;
+
+            background: linear-gradient(
+              160deg,
+              ${theme.accentStart} -20%,
+              ${theme.accentEnd} 60%
+            );
+            -webkit-background-clip: text;
+            -webkit-text-fill-color: transparent;
+          `}
+        >
+          1 : <Rate value={conversionRate} />
+        </h4>
+        <div
+          css={`
+            display: flex;
+            align-items: center;
+            margin-bottom: ${0.5 * GU}px;
+          `}
+        >
+          <h3
+            css={`
+              font-weight: ${fontWeight.medium};
+              font-size: 18px;
+            `}
+          >
+            Current conversion rate
+          </h3>
+          <Tooltip>
```

Comment:
For consistent design, use Aragon ui's Help component for the tooltip.  See this for example.
https://github.com/aragon/upgrade/blob/master/src/components/Migrate/Converter/ConversionRate.tsx#L128

Reply:
Thanks, was actually looking for that 😂